ptrace/binding/linux_struct.py

```diff
@@ -107,8 +107,14 @@ class user_regs_struct(Structure):
             ("dsisr", c_ulong),
             ("result", c_ulong),
         )
-    elif CPU_ARM:
+    elif CPU_ARM32:
         _fields_ = tuple(("r%i" % reg, c_ulong) for reg in range(18))
+    elif CPU_AARCH64:
+        _fields_ = tuple([*[("r%i" % reg, c_ulong) for reg in range(32)],
+                         ('sp', c_ulong),
+                         ('pc', c_ulong),
+                         ('pstate', c_ulong)]
+                         )
```

Comment:
I also found this in the kernel's code but would like confirmation on the format.